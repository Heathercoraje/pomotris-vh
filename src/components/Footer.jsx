import React, { Component } from 'react';
import green from '../../img/green.png'
import powderblue from '../../img/powderblue.png'

const Footer = () => (
  <footer>
    <img style={{ width: '3rem', margin: '1.5rem' }}  src={powderblue} />
    <p> Veredrec and Heathercoraje are in building process </p>
    <img style={{ width: '2rem', margin: '1.5rem'}}  src={green} />
	</footer>
);

export default Footer;
