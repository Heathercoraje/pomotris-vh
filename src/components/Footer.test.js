import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from './Footer';

describe('Footer', () => {
	const wrapper = shallow(<Footer />);
	it('renders correctly', () => {
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
	it('has footer element', () => {
		expect(wrapper.find('footer')).toHaveLength(1);
	});
});
