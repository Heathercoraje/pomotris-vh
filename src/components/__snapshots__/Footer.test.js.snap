// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer>
  <img
    src={
      Object {
        "process": [Function],
      }
    }
    style={
      Object {
        "margin": "1vh",
        "width": "5.5vh",
      }
    }
  />
  <p>
     Heathercoraje and Veredrec are in building process 
  </p>
  <img
    src={
      Object {
        "process": [Function],
      }
    }
    style={
      Object {
        "margin": "1vh",
        "width": "4vh",
      }
    }
  />
</footer>
`;
